src/api/immutable.js

```diff
@@ -79,6 +87,42 @@ class Reader extends helpers.NetworkObject {
     return lib.idata_size(this.app.connection, this.ref);
   }
 
+  /**
+   * Get the XOR-URL of the {@link ImmutableDataInterface}.
+   *
+   * @param {String} mimeType (experimental) the MIME type to encode in
+   * the XOR-URL as the codec of the content
+   * @returns {String}
+   * The XOR-URL of the ImmutableData.
+   * @example
+   * // Assumes {@link initialiseApp|SAFEApp} interface has been obtained
+   * const asyncFn = async () => {
+   *     try {
+   *         const cipherOpt = await app.cipherOpt.newPlainText();
+   *         const iDataWriter = await app.immutableData.create()
+   *         const data = 'Most proteins are glycosylated.
```

Comment:
For multiline text, I think ``` is cleaner that `'` (github is flagging this up as being an error, eg).

Reply:
let me take a look a this, and correct all docs that use`'` if it works with your suggestion